### ISSUE TITLE
Assets should be minified

### ISSUE THREAD

**Issue by eberkund** [modoboa/modoboa#753]
To decrease server requests and bandwidth all of the CSS and JS assets should be concatenated and minified.

This can be automated using Grunt or Gulp.

**Comment by tonioo** [comment]
Javascript dependencies are manged using bower so the minified versions are already there. But it is possible I do not include them in the templates...

**Comment by eberkund** [comment]
True, well if you are already using bower it shouldn't be much additional effort. I use Grunt on all my projects and this snippet works nicely for combining everything in to one minified file.

		concat: {
			dist: {
				src: [
					'assets/scripts/_source/jquery.*.js',
					'assets/scripts/_source/*.js'
				],
				dest: 'assets/scripts/combined.js'
			}
		},

		uglify: {
			build: {
				src: 'assets/scripts/combined.js',
				dest: 'assets/scripts/combined.min.js'
			}
		},
